rune/libcontainer/standard_init_linux.go

```diff
@@ -182,7 +183,24 @@ func (l *linuxStandardInit) Init() error {
 		if err != nil {
 			return err
 		}
-		return l.finalizeInit("/proc/self/exe", []string{"init-runelet", "enclave"})
+
+		if l.config.Config.Seccomp != nil && l.config.NoNewPrivileges {
+			if err := seccomp.InitSeccomp(l.config.Config.Seccomp); err != nil {
+				return newSystemErrorWithCause(err, "init seccomp")
+			}
+		}
+		exitCode, err := libenclave.StartInitialization()
+		if err != nil {
+			logrus.Debugf("enclave exitCode: %d", exitCode)
+			logrus.Fatal(err)
+			// Not returned as usual 1
+			// This error code should be returned to the parent process,
+			// it is passed to the user layer by layer
+			os.Exit(int(exitCode))
```

Comment:
Here should return 1 because having error means exitCode may be invalid, and thus returning it doesn't make sense.